src/main/kotlin/uk/gov/dluhc/printapi/config/SchedulerLockConfiguration.kt

```diff
@@ -11,7 +12,7 @@ import software.amazon.awssdk.services.dynamodb.DynamoDbClient
 @Configuration
 @EnableScheduling
 @ConditionalOnProperty("jobs.enabled", havingValue = "true")
-// @EnableSchedulerLock(defaultLockAtMostFor = "PT5M")
+@EnableSchedulerLock(defaultLockAtMostFor = "\${jobs.batch-print-requests.lock-at-most-for}")
 class SchedulerLockConfiguration(
```

Comment:
I think this property (Line 15) must be `global` as other jobs will also require similar lock. Something like this.

`@EnableSchedulerLock(defaultLockAtMostFor = "\${jobs.lock-at-most-for}")`